operator/k8s_cep_gc.go

```diff
@@ -64,14 +80,37 @@ func enableCiliumEndpointSyncGC() {
 							Errorf("Saw %T object while expecting *cilium_v2.CiliumEndpoint", cepObj)
 						continue
 					}
-
 					cepFullName := cep.Namespace + "/" + cep.Name
-					_, exists, err := watchers.PodStore.GetByKey(cepFullName)
-					if err != nil {
-						scopedLog.WithError(err).Warn("Unable to get pod from store")
-						continue
+
+					delCEP := once
+					if !once {
```

Comment:
Instead of having different branches about whether we should run this logic once, which complicates the flow, we can use `sync.Once` to ensure that it only runs once. This keeps the deleting logic the same, without having to complicate it to account for running once / running always. Have you considered this?

Reply:
The meaning of the variable `once` is different from the meaning of the `sync.Once`. the intention is to run the for loop for all endpoints one time, or not, depending the the GC is not supposed to be running or it is suppose to be running